index.d.ts

```diff
@@ -1668,6 +1672,54 @@ declare namespace Moleculer {
 		}
 	}
 
+	/**
+	 * you can use multiple modifier with a dot
+	 * e.g. `black.bgRed.bold`
+	 */
+	type KleurColor = keyof Kleur | string;
+
+	type ActionLoggerOptions = {
```

Comment:
objects should be modeled as `interface` rather than `type` because it allows for augmentation by plugins and consumers.

Reply:
not a problem for me but : 

this options can't be augmented (because middleware can't be augmented ? or I didn't known ? ), this options are totally dependends of the middleware ? ( also, this didn't change lot of things, so, not a problem to migrate to interfaces )